app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,20 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utills.JwtUtill;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 권한을 관리
+ */
```

Comment:
JavaDoc의 메인 디스크립션은 문장을 완성해 줍시다.
```suggestion
/**
 * 사용자 권한을 관리합니다.
 */
```